extensions/compose/src/compose-github-releases.spec.ts

```diff
@@ -66,63 +67,85 @@ test('expect grab 5 releases', async () => {
   expect(result.filter(release => release.label.includes('birthday')).length).toBe(0);
 });
 
-describe('Grab asset id for a given release id', async () => {
-  beforeEach(async () => {
-    // eslint-disable-next-line @typescript-eslint/consistent-type-imports
-    const fsActual = await vi.importActual<typeof import('node:fs')>('node:fs');
-
-    // mock the result of listReleaseAssetsMock REST API
-    const resultREST = JSON.parse(
-      fsActual.readFileSync(path.resolve(__dirname, '../tests/resources/compose-github-release-assets.json'), 'utf8'),
-    );
-
-    listReleaseAssetsMock.mockImplementation(() => {
-      return { data: resultREST };
+describe.each([
+  {
+    resource: '../tests/resources/compose-github-release-assets.json',
+    id: 91727807,
+    macOS_x86: 94785284,
+    macOS_arm: 94785273,
+    win_x86: 94785408,
+    win_arm: 94785398,
+    linux_x86: 94785376,
+    linux_arm: 94785298,
+  },
+  {
+    resource: '../tests/resources/compose-github-release-many-assets.json',
+    id: 91727807,
+    macOS_x86: 228416566,
+    macOS_arm: 228416557,
+    win_x86: 228416687,
+    win_arm: 228416679,
+    linux_x86: 228416660,
+    linux_arm: 228416575,
+  },
+])(
+  'Grab asset id for a given release id',
+  async ({ resource, id, macOS_x86, macOS_arm, win_x86, win_arm, linux_x86, linux_arm }) => {
+    beforeEach(async () => {
+      // eslint-disable-next-line @typescript-eslint/consistent-type-imports
+      const fsActual = await vi.importActual<typeof import('node:fs')>('node:fs');
+
+      // mock the result of listReleaseAssetsMock REST API
+      const resultREST = JSON.parse(fsActual.readFileSync(path.resolve(__dirname, resource), 'utf8'));
```

Comment:
You should be able to import the assets like `import ComposeRelease from '../resources/compose-github-release-many-assets.json`

Probably easier to import all using vite feature, than forcing the import of `node:fs`?